conf/backup-with-borg

```diff
@@ -17,12 +17,21 @@ filter_hooks() {
     sudo ls /usr/share/yunohost/hooks/backup/ /etc/yunohost/hooks.d/backup/ | grep "\-$1_" | cut -d"-" -f2 | uniq 2>> "$err_file"
 }
 
+healthchecks_url="$(sudo yunohost app setting "${borg_id}" healthchecks_url)"
+healthcheck() {
+    if [[ -n "$healthchecks_url" ]]; then
+        curl -s -m 10 -o /dev/null --data-raw "$(echo $2 | tail --bytes=10000)" "$healthchecks_url/$1"
```

Comment:
The API does not seem to support a trailing `/` when only the UUID is passed (i.e. `https://hc-ping.com/<uuid>/`).



```suggestion
        local signal="$1"
        # the expression "${signal:+/$signal}" adds a leading slash to the value of $signal 
        # only if the latter is not empty, otherwise it returns an empty string.
        local url="$healthchecks_url${signal:+/$signal}"
        curl -s -m 10 -o /dev/null --data-raw "$(echo $2 | tail --bytes=10000)" "$url"
```